docs/ui/AppearanceDemo.tsx

```diff
@@ -56,6 +69,16 @@ export const AppearanceDemo = ({
         visual style and dimensions of the component, available values are based
         on the active theme.
       </p>
+      {isVariantOrSizeMissing ? (
```

Comment:
I think this message should be part of the demo card. not be outside of it. Any yes, then it can not be a `SectionMessage` :D I think a hint text, that looks something like a help text is more appropiate here.

![Screenshot 2024-08-02 at 08 49 28](https://github.com/user-attachments/assets/ff434a39-4792-46bf-804d-c52205f78ab5)